### DISCUSSION TITLE
shuttermode 1 | shutterbutton1 ist broken in the newer versions

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#15848] [Q&A] [answered]
Hello . just set up a new flashed shelly 2.5; Button on L & SW1 
Template:
`{"NAME":"Shelly 2.5 (buttons)","GPIO":[320,0,0,0,224,129,0,0,640,128,608,225,3456,4736],"FLAG":0,"BASE":18}
`
Tasmota: Version greater them Tasmota 2022.01.3 von Theo Arends

```
switchmode1 3
so80 1
shuttermode 1
interlock 1,2
interlock ON
suttercloseduration 30
sutteropenduration 30
shuttersethalfway 63
```

everything it works fine .. so far

applying:
```
shutterbutton1 1 t 50 25 t - - - - 1
RSL: RESULT = {"ShutterButton1":"1 t 50 24 t - - - - 1"
```}

if u press the external button  one time -> Tasmota reboot
i tested 11,x 12.x ..
compiled with
```
#ifndef USE_SHUTTER
#define USE_SHUTTER     
#endif
#define MOTOR_STOP_TIME 750
```

switching back to  Tasmota 2022.01.3 it works as expected!!

Regards 
Mark_Muc
feel free to contact me on any further information!!

**Comment by Mark-Muc** [comment]
also i found a smal calc error

```
11:04:02.358 CMD: backlog shutterbutton1 1 t 50 25 t - - - - 1; shutterbutton1 2 t 50 25 t - - - - 1;
11:04:02.423 RSL: RESULT = {"ShutterButton1":"1 t 50 24 t - - - - 1"}
11:04:02.650 RSL: RESULT = {"ShutterButton1":"1 t 50 24 t - - - - 1|2 t 50 24 t - - - - 1"}
```
25 -> 24 in the result!!
Mark_Muc

**Comment by 001Tycoon** [comment, marked as answer]
I believe the error is the same as in https://github.com/arendst/Tasmota/issues/15825

**Reply by Mark-Muc** [reply]
ohh yes  001Tycoon you are right, didn't see this issue .
so this topic / thread can be closed!!
Mark_Muc

**Reply by Jason2866** [reply]
fixed with commit https://github.com/arendst/Tasmota/commit/ea226228f57e114a4a0da38ed82375dedb944691
